### DISCUSSION TITLE
translation into Brazilian Portuguese

### DISCUSSION THREAD

**Discussion by idalx** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1644] [General]
I want to be able to try to help you with the translation into Brazilian Portuguese, but I don't know Java or any other programming language, if possible, could you teach me the basics so I can try to translate. I don't know how to speak completely correct English, but I want to be able to try to help develop this mod that I played years ago and loved

**Comment by Millymilo00** [comment]
I think to translate this mod you need to duplicate any `.lang` file in the [`src/main/resources/assets/hbm/lang`](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/tree/master/src/main/resources/assets/hbm/lang) directory (The English `.lang` file is probably the best to duplicate, because it is the most up to date file, but if you know any of the other languages better than English that have a `.lang` file, you can use those as a base instead). Then name the new file `pt_BR.lang`, and edit it to translate all the text after the ids.
